app/views/passengers/show.haml

```diff
@@ -35,7 +35,7 @@
     = @passenger.rides_expire.strftime('%m/%d/%Y')
 %p
   %b Registering Dispatcher:
-  = @passenger.registerer || 'Unknown'
+  = @passenger.registerer ? @passenger.registerer.name : 'Unknown'
```

Comment:
Here's a good place to use `try` or the [safe navigation operator](http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/)